sample.env

```diff
@@ -108,8 +95,8 @@ MAPBOX_INIT_ZOOM="10"
 GCP_PROJECT_ID = ''

```

Comment:
This one wasn't changed here, but I don't seem to have it in my existing .env file - do we know if it's needed for anything?

Reply:
Looks like this is used in docker-compose-addons.yml, I don't see any documentation for it...

Reply:
This is required for the jpo_iss_health_check image. It pulls fron gcp secrets manager